cmake/modules/SwiftImplicitImport.cmake

```diff
@@ -6,7 +6,7 @@ function(swift_supports_implicit_module module_name out_var)
       "${CMAKE_Swift_COMPILER}"
       -Xfrontend -disable-implicit-${module_name}-module-import
       -Xfrontend -parse-stdlib
-      -c - -o /dev/null
+      -parse -
```

Comment:
Linux `swiftc` reads the output file after compiling (for e.g. autolink-extract). Since `/dev/null` is not a valid object file, this failed.
All we want to check here is if the frontend accepts the command line option, so `-parse` should be enough.